templates/organization/package/details.html.twig

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html.twig' %}
+
+{% block header %}{{ package.name }} details{% endblock %}
+{% block header_btn %}
+    <a href="{{ path('organization_packages', {"organization":organization.alias}) }}" class="btn btn-primary">
+        {% include 'svg/package.svg' %} Package list
+    </a>
+{% endblock %}
+
+{% block content %}
+
+    {% if package.name %}
+        <h4>Package name</h4>
+        <p>{{ package.name }}</p>
+
+        <h4>Description</h4>
+        <p>{{ package.description }}</p>
+
+        {% if package.latestReleaseDate %}
+            <h4>Latest version</h4>
+            <p>{{ package.latestReleasedVersion }}</p>
+
+            <h4>Released</h4>
+            <p>
+                <span title="{{ package.latestReleaseDate | date('Y-m-d H:i:s') }}">{{ package.latestReleaseDate | time_diff }}</span>
+            </p>
+
+        {% endif %}
+
+        <h3>Available versions</h3>
+
+        <table class="table">
+            <thead>
+            <tr>
+                <th>Version</th>
+                <th>Released</th>
+                <th>Reference</th>
+                <th></th>
```

Comment:
Maybe we can add `size` column here?

Reply:
Any idea how I can get the size easily? I could add a ->size to `distStorage` that returns the size of the archive?

Reply:
Sounds good: `Storage -> size(Dist $dist): int`